src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -560,6 +589,11 @@ export default class AircraftController {
             aircraftModel.fms.departureRunwayModel.addAircraftToQueue(aircraftModel.id);
             aircraftModel.setFlightPhase(FLIGHT_PHASE.WAITING);
             aircraftModel.shouldTakeOffWhenRunwayIsClear = true;
+
+            if (runwayCommands && runwayCommands[aircraftModel.fms.departureRunwayModel.name]) {
+                const commandString = runwayCommands[aircraftModel.fms.departureRunwayModel.name];
+                this._runCommandOnNewAircraft(aircraftModel, commandString);
```

Comment:
missing new line between assignment and method call

Reply:
done